src/main/java/marquez/db/JobDao.java

```diff
@@ -38,8 +37,7 @@
   public void insert(@BindBean Job job);
 
   @SqlUpdate("UPDATE jobs SET current_version_guid = :version_guid WHERE guid = :job_guid")
-  public void setCurrentVersionGuid(
-      @Bind("job_guid") UUID jobGuid, @Bind("version_guid") UUID currentVersionGuid);
+  public void setCurrentVersionGuid(UUID jobGuid, UUID currentVersionGuid);
```

Comment:
When binding method arguments to the SQL statement in ` @SqlUpdate` it's important to note that `@Bind` does allow for overriding what argument name it's bound to. That is, `@Bind("job_guid")` will bind `job_guid` in the SQL statement (below) to `:job_guid` instead of `jobGuid`:

```sql
UPDATE jobs SET current_version_guid = :version_guid WHERE guid = :job_guid
```
Knowing that, this is way the tests are failing. We need to update the argument name referenced in the SQL statement with that name of the parameter method.

```sql
UPDATE jobs SET current_version_guid = : currentVersionGuid WHERE guid = :jobGuid
```

see: http://jdbi.org/#_arguments